dev/drivers/scripts/analyses/stats/jevs_analyses_rtma_grid2obs_stats.sh

```diff
@@ -15,7 +15,7 @@ cd $PBS_O_WORKDIR
 
 export model=evs
 
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_analyses_EE2/EVS
```

Comment:
Can you revert this HOMEevs definition?

Reply:
Nerts, I thought I took care of that!

Perry